### DISCUSSION TITLE
Question About Autocomplete Results

### DISCUSSION THREAD

**Discussion by gusdemayo** [Smile-SA/elasticsuite#2477] [Q&A]
**Is your question related to a problem? Please describe.**
A clear and concise description of what the problem is. 

We've noticed that the results in our autocomplete better reflect the results we want to show in our full product search, is it expected for results in the autocomplete to be different than in the full search?

**Additional context**
Add any other context or screenshots about your question here.

**Comment by romainruaud** [comment]
How the autocomplete work is explained in details in https://github.com/Smile-SA/elasticsuite/issues/355#issuecomment-286831041 and https://github.com/Smile-SA/elasticsuite/issues/1277#issuecomment-456385530

**Reply by gusdemayo** [reply]
Ok, thank you, that all makes sense, but I have kind of an additional/follow up question - 

On our site (www.midlandsci.com), when we search for the term "Acetonitrile" we get a different set of results in the autocomplete than on the search page, which makes sense per the linked issues above.

Is there a way we could get results closer to the autocomplete results on the search page?

**Reply by amenk** [reply]
The page is broken currently (does not load CSS).
Which sorting do you have on the result page? Can you provide some screenshots of the differences? It is possible to define different optimizers for autocomplete and search page -> can you check those?
